codes/rust/chapter_array_and_linkedlist/linked_list.rs

```diff
@@ -73,7 +74,7 @@ fn main() {
 
     /* 插入节点 */
     insert(&n0, ListNode::new(0));
-    print!("插入节点后的链表为 ");
+    print!("插入节点0后的链表为 ");
```

Comment:
Do not modify the comments. They are consistent across different programming languages.

Reply:
got you